### ISSUE TITLE
CLI "peer" command would be better called "start"

### ISSUE THREAD

**Issue by lehors** [hyperledger-archives/fabric#1158]
The CLI commands are:
      peer        Runs the peer.
      status      Returns status of the peer.
      stop        Stops the running peer.
This means that to start a peer one has to type: "./peer peer" and we have the pair "peer, stop" to start and stop the peer. This is not intuitive.

I suggest instead we have the commands:
      start        Starts the peer.
      status      Returns status of the peer.
      stop        Stops the running peer.
Which would be much more intuitive.

**Comment by lehors** [comment]
I'd be happy to submit a PR to make the change if others agree with the change.

**Comment by christo4ferris** [comment]
SGTM

**Comment by jeffgarratt** [comment]
The executable has the dual purpose of acting as the Peer, as well as the CLI.  Since we have now renamed the executable 'peer', it may be prudent to rename the 'peer' command to 'server', with several subcommands of 'start','stop', etc.  In this way the now 'server' related commands (or CLI) may feel more segregated.

for example, starting the peer would be...

> peer server start
>   And the other commands would be readily identifiable as CLI based.

Thoughts?  @lehors @binhn

**Comment by christo4ferris** [comment]
@jeffgarratt other than 'server' what other targets would be invoked in that manner?

**Comment by jeffgarratt** [comment]
@christo4ferris there is also 'chaincode', which supports several sub-commands ('deploy', 'invoke', 'query').

for example, to deploy a chaincode...

> peer chaincode deploy

**Comment by binhn** [comment]
The list of commands:

```
Available Commands: 
  peer        Runs the peer.
  status      Returns status of the peer.
  stop        Stops the running peer.
  login       Logs in a user on CLI.
  network     Lists all network peers.
  chaincode   chaincode specific commands.
  help        Help about any command
```

I ageed with @jeffgarratt that `start` and `stop` impact the target peer, but the remaining ones are CLI and some with subcommands or parameters. The online help is not very complete, which we should also cover in this issue that @lehors can take care of.

**Comment by jeffgarratt** [comment]
@lehors @binhn @christo4ferris 

I would think something like the following would be inutitive:

```
peer
   help
   server
      start
      stop
      status
   chaincode
      deploy
      invoke
      query
   network
      login
      list
```

So...
to start server:  peer server start

to login:  peer network login

Seems to read well as you type the command.  Could also consider changing 'server' to 'node' if the community is more use to that nomenclature.

**Comment by lehors** [comment]
@jeffgarratt I have to admit I wasn't convinced at first but your latest proposal looks quite attractive to me. :-)
I do think node sounds better than server in this context.

**Comment by smithbk** [comment]
Since the executable is dual purpose (peer and cli), what about renaming it to something more generic like "hl" (for "hyperledger")?

**Comment by lehors** [comment]
@smithbk I had the same thought. fabric would make sense then. But I saw an old reference to that name in a yaml file so I wonder if this was considered and rejected for some reason. @jeffgarratt @binh, do you know?

Side note: @smithbk your profile doesn't give much of a clue as to who you are. In an effort to promote transparency people are encouraged to disclose their name, affiliation, etc. So, please, consider doing so.
Thanks.

**Comment by binhn** [comment]
We didn't like fabric, hl or hyperledger is not appropriate. We're been using peer since beginning, so we stick with it. Jeff's proposal sounds good to me

**Comment by lehors** [comment]
@binhn, does that mean you prefer server over node? node is shorter to type and is used in some places but of course it's a server so both are reasonable proposals.
Your call.

**Comment by lehors** [comment]
Addressed by PR #1357
